server/services/utils/formatForView.js

```diff
@@ -32,6 +32,8 @@ function formatObjectForViewWithOptions(object, options) {
 function formatAgencyLocationDesc(agencyLocationDesc) {
     if (agencyLocationDesc && agencyLocationDesc.toLowerCase().indexOf('(hmp)') !== -1) {
         return 'HMP ' + setCase.capital(agencyLocationDesc.toLowerCase().replace('(hmp)', '')).trim();
+    } else if (agencyLocationDesc && agencyLocationDesc.toLowerCase().indexOf('hmp') !== -1) {
+        return 'HMP ' + setCase.capital(agencyLocationDesc.toLowerCase().replace('hmp', '')).trim();
     }
```

Comment:
This is all abit WIP until we know more about exactly what the nomis responses look like